backend-python/README.md

```diff
@@ -0,0 +1,28 @@
+# Python FastAPI Backend
+### Features
+- [x] FastAPI
+- [x] SQLAlchemy
+- [x] Poetry
+- [x] Prospector
+- [x] Flyway
+- [x] Docker
+- [x] Docker Compose
+- [x] GitHub Actions
+
+## Getting Started
+
+### Prerequisites
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Local Development
+- Run the `docker-compose -f .\docker-compose.py.yml up` command to start the entire stack.
```

Comment:
Slash typo, should be a forward slash: `-f ./docker-compose.py.yml`.

Reply:
I will create another issue and fix it ;)